mwdblib/object.py

```diff
@@ -387,6 +387,17 @@ def remove_attribute(self, attribute_id: int) -> None:
         )
         self._expire("attributes")
 
+    @APIClient.requires("2.6.0")
```

Comment:
Comment delete is available from the first public version (v2) so it's not necessary
```suggestion
```